### PR TITLE
Open product image test case was implemented

### PR DESCRIPTION
**Pull request by GoshkaEgoshka**
In product_page file added open_product_image function
Added oct/tests/web/open_prod_img.py for run test
In brand page was deleted mistakes

### CODE REVIEW

File: oct/pages/product_page.py [hunk 1]
```diff
@@ -23,3 +23,10 @@ def open_brand_page(self) -> None:
             '//*[@id="content"]/div[1]/div[2]/ul[1]/li[1]/a'
         )
         brand_name.click()
+
+    def open_product_image(self) -> bool:
```
Comment (extsoft) [inline oct/pages/product_page.py]: @GoshkaEgoshka What does this method do?

Comment (GoshkaEgoshka) [reply oct/pages/product_page.py]: @extsoft This method checks if the product picture opens if you click on it.

Comment (extsoft) [reply oct/pages/product_page.py]: @GoshkaEgoshka the you need to have tow methods:
1. open an image (click on it)
2. check if image is open

This method violates the single responsibility principle.

Comment (GoshkaEgoshka) [reply oct/pages/product_page.py]: @extsoft I understood

Comment (GoshkaEgoshka) [reply oct/pages/product_page.py]: @extsoft Done!

File: oct/pages/product_page.py [hunk 2]
```diff
@@ -23,3 +23,12 @@ def open_brand_page(self) -> None:
...
         brand_name.click()
+
+    def open_product_image(self) -> None:
+        product_img = self._browser.find_element_by_xpath(
+            '//*[@id="content"]/div[1]/div[1]/ul[1]/li[2]/a/img'
+        )
+        product_img.click()
+
+    def is_open_image(self) -> bool:
+        return self._browser.find_element_by_class_name("mfp-img")
```
Comment (extsoft) [inline oct/pages/product_page.py]: @GoshkaEgoshka An exception will be thrown if the element is not found. So, this method can return True or thrown an error. But I think if you can't find an element, you have to return False.

Comment (GoshkaEgoshka) [reply oct/pages/product_page.py]: @extsoft I think I solved  this problem. Please, check.

File: oct/pages/product_page.py [hunk 3]
```diff
@@ -23,3 +24,15 @@ def open_brand_page(self) -> None:
...
+
+    def open_product_image(self) -> None:
+        product_img = self._browser.find_element_by_xpath(
+            '//*[@id="content"]/div[1]/div[1]/ul[1]/li[2]/a/img'
+        )
+        product_img.click()
+
+    def is_open_image(self) -> bool:
+        try:
+            return self._browser.find_element_by_class_name("mfp-img")
```
Comment (extsoft) [inline oct/pages/product_page.py]: @GoshkaEgoshka you need `return True` here, not a `WebElement`.

Comment (GoshkaEgoshka) [reply oct/pages/product_page.py]: @extsoft I made a change. Please, take a look.

**Pull request merged**